erpnext/patches/v13_0/update_item_master_doctype_field.py

```diff
@@ -0,0 +1,7 @@
+import frappe
+
+def execute():
+	frappe.reload_doc("stock", "doctype", "item")
+	items = frappe.db.get_all('Item', fields = ['name', 'inspection_required_before_manufacturing'])
```

Comment:
Add a `frappe.db.column_exists` before you do the `get_all`. We have some databases that don't even have this field.